components/user/card-activity.vue

```diff
@@ -35,9 +41,15 @@
 				<div v-if="data.state" class="block overflow-hidden text-ellipsis whitespace-nowrap">
 					{{ data.state }}
 				</div>
-				<div v-if="elapsed" class="block overflow-hidden text-ellipsis whitespace-nowrap">{{ elapsed }} elapsed</div>
+				<div v-if="elapsed && !spotifyImageUrl" class="block overflow-hidden text-ellipsis whitespace-nowrap">{{ elapsed }} elapsed</div>
 			</div>
 		</div>
+		<template v-if="spotifyImageUrl">
+			<user-card-seekbar 
+				:start-time-ms="data.timestamps!.start!"
+				:end-time-ms="data.timestamps!.end!"
+			/>
+		</template>
```

Comment:
Conditional `<template>`s are useful for showing more elements, otherwise, it can be inlined for simplicity:
```suggestion
		<user-card-seekbar
			v-if="spotifyImageUrl"
			:start-time-ms="data.timestamps!.start!"
			:end-time-ms="data.timestamps!.end!"
		/>
```

Reply:
I thought that was wrong. Corrected.